packs/journals/archetypes.json

```diff
@@ -208,7 +208,7 @@
dium.pf2e.feats-srd.Item.Nature's Precision] <span style=\"float:right\">Feat 6</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.6B7ksTtpbfp114sA inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Swift Guardian] <span style=\"float:right\">Feat 6</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.tOtrtDR7ftHhPFm4 inline hr=false]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Tenacious Endurance] <span style=\"float:right\">Feat 6</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.LT5rw7JfemZSXUKB inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Incredible Beastmaster Companion] <span style=\"float:right\">Feat 8</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.aEws1NR19Lbu1Kio inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Pack Movement] <span style=\"float:right\">Feat 8</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.QBOhIDAIvpHUGSji inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Beastmaster Bond] <span style=\"float:right\">Feat 10</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.3R09Hl6IDMgPcSs0 inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Ferocious Charge] <span style=\"float:right\">Feat 10</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.1VLIJgnTUxNlf7T8 inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Running Kick] <span style=\"float:right\">Feat 10</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.NfNCSLCNmLZZyGqU inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Sinking Jaws] <span style=\"float:right\">Feat 10</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.obqFjbH0fciMH5F9 inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Sweeping Tail] <span style=\"float:right\">Feat 10</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.RBvmQJ661GpT5lkt inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Vicious Rend] <span style=\"float:right\">Feat 10</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.NtWM5UgKBYvrTLQ4 inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Beastmaster's Call] <span style=\"float:right\">Feat 12</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.5kUQuIP8N57MXhuz inline hr=false]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Billowing Wings] <span style=\"float:right\">Feat 12</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.um6NtNmEpWxGelom inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Specialized Beastmaster Companion] <span style=\"float:right\">Feat 14</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.bki36RiEM5FR4aiT inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Heightened Instincts] <span style=\"float:right\">Feat 18</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.WlFJyOz73m67HmW2 inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Lead the Pack] <span style=\"float:right\">Feat 16</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.Aeea44xyJqYArFun inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Pack Takedown] <span style=\"float:right\">Feat 20</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.ZeQP0awEkOHWmMRa inline hr=false]</p>",
```

Comment:
```suggestion
                "content": "<p>You attract the loyalty of animals, and as your powers increase, you can command more of them, briefly inhabit their body to perceive what they perceive, and even communicate with them over vast distances.</p><p><strong>Additional Feats</strong> <strong>4th</strong> @UUID[Compendium.pf2e.feats-srd.Item.Animal Empathy (Ranger)],@UUID[Compendium.pf2e.feats-srd.Item.Initiate Warden](@UUID[Compendium.pf2e.spells-srd.Item.Heal Companion] and @UUID[Compendium.pf2e.spells-srd.Item.Magic Hide] only); <strong>6th</strong> @UUID[Compendium.pf2e.feats-srd.Item.Advanced Warden] (@UUID[Compendium.pf2e.spells-srd.Item.Enlarge Companion] only), @UUID[Compendium.pf2e.feats-srd.Item.Companion's Cry]; <strong>14th</strong> @UUID[Compendium.pf2e.feats-srd.Item.Side by Side (Ranger)]{Side by Side}. Warden spells granted by these feats are beastmaster focus spells for you.</p><p> </p><table border=\"1\" class=\"pf2e remaster\"><thead><tr style=\"text-align:right\"><th>Beastmaster Animal Companions</th></tr></thead><tbody><tr><td>If you're playing a beastmaster, you determine the statistics and abilities of your animal companions according to the rules on pages 206–211 of <em>Player Core</em>. As a beastmaster, it's possible for you to have more than one animal companion at one time—up to four companions—but only one of those companions, your \"active companion,\" follows you during exploration and in encounters; the rest are nearby, usually foraging or hunting for food. As soon as you gain a second animal companion from the Beastmaster archetype, you also gain @UUID[Compendium.pf2e.actionspf2e.Item.Call Companion] to switch your active companion. These rules apply to all your companions, regardless of whether you got the animal companion from the beastmaster archetype or from another source.</td></tr></tbody></table><h2>@UUID[Compendium.pf2e.feats-srd.Item.Beastmaster Dedication] <span style=\"float:right\">Feat 2</span></h2><p><strong>Prerequisites</strong> trained in Nature</p><hr /><p>You gain the service of a young animal companion that travels with you and obeys your commands. Contrary to the usual rules for animal companions, this feat can grant you a second animal companion. If you ever have more than one animal companion, you gain the Call Companion action. See the Beastmaster Animal Companions sidebar for details on this action.</p><p>Certain beastmaster feats give you focus spells. When you gain your first beastmaster focus spell, you become trained in the spell attack modifier and spell DC statistics. Your key spellcasting attribute for beastmaster archetype spells is Charisma, and they are primal spells. You can Refocus by feeding, playing with, or otherwise tending to an animal companion.</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Additional Companion] <span style=\"float:right\">Feat 4</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.zBLrZE5aCkpaTK2N inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Spirit of the Beast] <span style=\"float:right\">Feat 4</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.GTKzA1rfUtQwxvgz inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Mature Beastmaster Companion] <span style=\"float:right\">Feat 4</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.uCElsebJ45ltmZMT inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Beastmaster's Trance] <span style=\"float:right\">Feat 6</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.Ckglzh4dXcGWPNS3 inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Bestial Protection] <span style=\"float:right\">Feat 6</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.bHHBEVmrY61MIgsP inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Nature's Precision] <span style=\"float:right\">Feat 6</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.6B7ksTtpbfp114sA inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Swift Guardian] <span style=\"float:right\">Feat 6</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.tOtrtDR7ftHhPFm4 inline hr=false]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Tenacious Endurance] <span style=\"float:right\">Feat 6</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.LT5rw7JfemZSXUKB inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Incredible Beastmaster Companion] <span style=\"float:right\">Feat 8</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.aEws1NR19Lbu1Kio inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Pack Movement] <span style=\"float:right\">Feat 8</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.QBOhIDAIvpHUGSji inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Beastmaster Bond] <span style=\"float:right\">Feat 10</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.3R09Hl6IDMgPcSs0 inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Ferocious Charge] <span style=\"float:right\">Feat 10</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.1VLIJgnTUxNlf7T8 inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Running Kick] <span style=\"float:right\">Feat 10</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.NfNCSLCNmLZZyGqU inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Sinking Jaws] <span style=\"float:right\">Feat 10</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.obqFjbH0fciMH5F9 inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Sweeping Tail] <span style=\"float:right\">Feat 10</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.RBvmQJ661GpT5lkt inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Vicious Rend] <span style=\"float:right\">Feat 10</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.NtWM5UgKBYvrTLQ4 inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Beastmaster's Call] <span style=\"float:right\">Feat 12</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.5kUQuIP8N57MXhuz inline hr=false]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Billowing Wings] <span style=\"float:right\">Feat 12</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.um6NtNmEpWxGelom inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Specialized Beastmaster Companion] <span style=\"float:right\">Feat 14</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.bki36RiEM5FR4aiT inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Heightened Instincts] <span style=\"float:right\">Feat 18</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.WlFJyOz73m67HmW2 inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Lead the Pack] <span style=\"float:right\">Feat 16</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.Aeea44xyJqYArFun inline]</p><h2>@UUID[Compendium.pf2e.feats-srd.Item.Pack Takedown] <span style=\"float:right\">Feat 20</span></h2><p>@Embed[Compendium.pf2e.feats-srd.Item.ZeQP0awEkOHWmMRa inline hr=false]</p>",
```

Reply:
Yeah, I see what you mean about it not allowing you to commit.